README.md

```diff
@@ -98,7 +98,9 @@ Additional Commands:
 
 Flags:
   -C, --ci            Disable realtime terminal updates and color suitable for CI and scripting (default false)
-  -F, --from string   Comma-separated list of location values to match against or measurement ID. For example the partial or full name of a continent, region (e.g eastern europe), country, US state, city or network (default "world"). (default "world")
+  -F, --from string   Comma-separated list of location values to match against or a measurement ID
+                      For example, the partial or full name of a continent, region (e.g eastern europe), country, US state, city or network
+                      Or use [@1 | first, @2 ... @-2, @-1 | last] to run with the probes from previous measurements. (default "world")
```

Comment:
Lets document that last and previous are both supported. In some cases "last" can be confusing and previous is clearer.

Reply:
And can you please add a few examples of @1 and @-2 uses to the appropriate section?